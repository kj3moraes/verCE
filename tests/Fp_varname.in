3fja = 000.1