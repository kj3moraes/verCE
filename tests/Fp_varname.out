NUM:3 
ID:fja 
BECOMES:= 
NUM:000.1 
